tests/test_model.py

```diff
@@ -566,3 +588,94 @@ class Child(Base):
     assert len(schema.columns) == 2
     assert schema.columns.get("_a_child", None) is not None
     assert schema.columns.get("_b", None) is not None
+
+
+def test_inherit_override():
```

Comment:
We can merge this test with the one above `test_inherit_alias`.

Reply:
I tried to systematically cover all cases (omitted, present, alias) and their combinations in one test.